sql/util_time.sql

```diff
@@ -46,3 +57,31 @@ RETURNS INT8 AS '@MODULE_PATHNAME@', 'ts_continuous_agg_watermark_materialized'
 
 CREATE OR REPLACE FUNCTION _timescaledb_functions.subtract_integer_from_now( hypertable_relid REGCLASS, lag INT8 )
 RETURNS INT8 AS '@MODULE_PATHNAME@', 'ts_subtract_integer_from_now' LANGUAGE C STABLE STRICT;
+
+-- Convert integer UNIX timestamps in microsecond to a timestamp range.
+CREATE OR REPLACE FUNCTION _timescaledb_functions.as_multirange(
+    base tstzrange, low_usec bigint, high_usec bigint
```

Comment:
There are a mismatch between the signature here and in latest-dev.sql

Reply:
Fixed.